### ISSUE TITLE
Impossible to connect

### ISSUE THREAD

**Issue by frafor** [adamjimenez/shiftedit#381]
Error: please try again!Original error message: User profile request failed! Google returned an invalid response:(object) array( 'error' => (object) array( 'code' => 403, 'message' => 'Legacy People API has not been used in project 471018772965 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/legacypeople.googleapis.com/overview?project=471018772965 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.', 'status' => 'PERMISSION_DENIED', 'details' => array ( 0 => (object) array( '@type' => 'type.googleapis.com/google.rpc.Help', 'links' => array ( 0 => (object) array( 'description' => 'Google developers console API activation', 'url' => 'https://console.developers.google.com/apis/api/legacypeople.googleapis.com/overview?project=471018772965', ), ), ), 1 => (object) array( '@type' => 'type.googleapis.com/google.rpc.ErrorInfo', 'reason' => 'SERVICE_DISABLED', 'domain' => 'googleapis.com', 'metadata' => (object) array( 'consumer' => 'projects/471018772965', 'service' => 'legacypeople.googleapis.com', ), ), ), ), )missing email

That's the response when I try to connect using Google.

Using github:

Something went wrong
The webmaster has been notified.

(And also with the others)

So, basically I can not use shiftedit anymore

**Comment by frafor** [comment]
![image](https://user-images.githubusercontent.com/4719577/117878903-e3eae900-b26b-11eb-98c7-a8b02289895b.png)

And that's what happens when I try to re-register.

**Comment by adamjimenez** [comment]
Hi,

Google and Facebook login are fixed. I'm working on the others.

**Comment by adamjimenez** [comment]
ok, should all be fixed now. Let me know if you have any more issues.

**Comment by frafor** [comment]
Thanks! Yes, it has been fixed. However, do things like this one happen often?

**Comment by adamjimenez** [comment]
The platform was upgraded to PHP 8 which led to a few issues. We should be good for a while now.
